bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/BigtableChannels.java

```diff
@@ -87,87 +87,66 @@ public static CloseableChannel createChannel(TransportOptions transportOptions,
    * <p> To create a pool, createPool has to be true and channel count has to be greater than one.
    * </p>
    */
-  public static CloseableChannel createChannel(final TransportOptions transportOptions,
+  public static Channel createChannel(final TransportOptions transportOptions,
       final ChannelOptions channelOptions, final ExecutorService executor, boolean createPool) {
     Preconditions.checkArgument(
         transportOptions.getTransport() == TransportOptions.BigtableTransports.HTTP2_NETTY_TLS,
         "Bigtable requires the NETTY_TLS transport.");
 
-    int channelCount = channelOptions.getChannelCount();
+    int channelCount = createPool ? channelOptions.getChannelCount() : 1;
```

Comment:
I moved this setting to Transport options.  Only the data api needs more than 1 channel in the pool.